data_sources.py

```diff
@@ -8,7 +8,7 @@
                  ],
                  "Liberecky": [
                               "U2HRAD01_T_N.csv.zip",
-			      "U2STRR01_T_N.csv.zip",
+                               "U2STRR01_T_N.csv.zip",
```

Comment:
Sice to odsazení v Libereckém kraji, není pěkně zarovnané, ale commit dat pardubickych by sem neměl zasahovat.
Vyladění odsazení pak můžeme udělat samostatným commitem.